### DISCUSSION TITLE
new connection when restarting process

### DISCUSSION THREAD

**Discussion by muzzol** [asternic/wuzapi#46] [General]
Hi,

I'm testint wuzapi so sorry if these questions are basic.

I connected once stopped the process, did some changes and when I started again with **./wuzapi -wadebug DEBUG** the connection was lost.

do i need to connect everytime?

how can I do that unnatended (headless server)?

**Comment by asternic** [comment]
The whatsapp sessions are saved, it should reconnect automatically when you restart.
